lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -285,15 +285,15 @@ defmodule ExUnit.Runner do
   defp spawn_test(config, test, context) do
     parent_pid = self()
     timeout = get_timeout(config, test.tags)
-    {test_pid, test_ref} = spawn_test_monitor(config, test, parent_pid, context)
+    {test_pid, test_ref} = spawn_test_monitor(test, parent_pid, context, config.seed)
```

Comment:
I'm 👎 for this change. I liked better when `spawns_test/3` did not know that the only thing that `spawn_test_monitor/4` needs is the seed configuration.

Reply:
yeah, it's understandable.
the original idea was the replace generate_test_seed, which only needs the seed.